src/com/amazon/corretto/crypto/provider/Loader.java

```diff
@@ -52,10 +51,10 @@ static InputStream getTestData(String fileName) {
             throw new IllegalArgumentException("Invalid filename: " + fileName);
         }
         final InputStream result = AccessController.doPrivileged(
-                (PrivilegedAction<InputStream>) () -> Loader.class.getResourceAsStream("/test-data/" + fileName)
+                (PrivilegedAction<InputStream>) () -> Loader.class.getResourceAsStream("testdata/" + fileName)
```

Comment:
Please fix JavaDoc on this method.

Reply:
I saw the CMakeList.txt and I got a bit confused, as that specific variable  seems to point to this directory:
https://github.com/corretto/amazon-corretto-crypto-provider/tree/develop/test-data

Thile the one I've moved is:
https://github.com/corretto/amazon-corretto-crypto-provider/tree/develop/extra-jar-files/test-data

Although both folders have the same names, they do not have the same contents.

Regarding the javadoc, I'll fix it

Reply:
Hmm, don't move it then. The CMAKE variable is for test data we don't currently ship in the JAR. I got confused.